datalad/distribution/add_sibling.py

```diff
@@ -99,17 +91,11 @@ class AddSibling(Interface):
                 This option is ignored if there is already a configured sibling
                 dataset under the name given by `name`""",
             constraints=EnsureStr() | EnsureNone()),
-        recursive=recursion_flag,
-        fetch=Parameter(
-            args=("--fetch",),
-            action="store_true",
-            doc="""fetch the sibling after adding"""),
```

Comment:
hm, so you are suggesting to remove recursive mode of operation and autofetching upon adding... I think I have used both of those